server/config.py

```diff
@@ -0,0 +1,17 @@
+import redis
+
+# flake8: noqa
+
+
+class Config:
+    # environ.get('FLASK_KEY')
+    SECRET_KEY = 'g+hH\xebiv\x9f\x8e\xb8\x95\xed?]>\x81,\x1c\\\xc1\x0fm\x95\x96'
```

Comment:
should this be on github?

Reply:
@Tengelma is it okay for this key to be here?

Reply:
It's mainly used to keep sessions secure. I think its OK in the near term but once we properly deploy the site we'll want to pull it from some local file that contains it.

Basically just need something in our startup shell script that says

```
if SECRET_KEY file doesn't exist:
  generate a key
  save secret key to file SECRET_KEY
set FLASK_KEY environment variable to value in file
```

and then grab that value from ENV when starting the python server. Would add that file to our .gitignore, obviously.

Reply:
Sounds good.

Reply:
Should I make an issue to remind ourselves of this?